experiment/measurer.py

```diff
@@ -578,7 +578,10 @@ def set_up_coverage_binaries(pool, experiment):
     ]
     coverage_binaries_dir = build_utils.get_coverage_binaries_dir()
     if not os.path.exists(coverage_binaries_dir):
-        os.makedirs(coverage_binaries_dir)
+        try:
+            os.makedirs(coverage_binaries_dir)
+        except Exception as err:
+            logger.warning(str(err))
```

Comment:
What happens if you don't catch the exception?
I'm fine with not logging anything since this seems to be expected.

Reply:
If there is not catching, the measure_loop process will end and then there will be no measurement and no report generation.